stdlib/string.ml

```diff
@@ -228,14 +229,15 @@ let hash x = seeded_hash 0 x
 (* duplicated in bytes.ml *)
```

Comment:
Please duplicate the same change in `bytes.ml`.

Reply:
Bytes.t are mutable, so we probably want to always copy the string ? Maybe the comment should be edited ?